pkg/scannerv4/mappers/mappers.go

```diff
@@ -589,6 +578,16 @@ func toProtoV4VulnerabilitiesMap(
 				Str("nvd_published", nvdVuln.Published).
 				Msg("issued time invalid: leaving empty")
 		}
+		var vulnEPSS *epss.EPSSItem
```

Comment:
not sure if I told you to move this before or someone else, but I actually think it's ok in the original position since it's pretty close to where `foundCVE` is defined